cgi-bin/LJ/HTMLControls.pm

```diff
@@ -224,8 +224,10 @@ sub _html_option {
 
     # are there additional data-attributes?
     my $data_attribute = '';
-    if ( $item->{data} ) {
-        $data_attribute .= " data-$_='$item->{data}->{$_}'" foreach keys %{$item->{data} || {}};
+    my %item_data = $item->{data} ? %{$item->{data}} : ();
```

Comment:
I don't know if it matters here, but traditionally if we're about to deref something we do a ref check:

```my %item_data = ref $item->{data} eq 'HASH' ? %{$item->{data}} : ();```